test/storybook/visual-regression/focus.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect, Page, Locator } from "@playwright/test"
+
+const goTo = async (page: Page, slug: string) => {
+  await page.goto(`/iframe.html?id=meta-focus--${slug}`)
+}
+
+const expectSnapshot = async (name: string, elem: Locator) => {
+  expect(await elem.screenshot()).toMatchSnapshot({ name: `${name}.png` })
+}
+
+const allSlugs = ["slim-transparent", "slim-filled", "bold-filled", "colored"]
+
+test.describe.configure({ mode: "parallel" })
+
+test.describe("Focus", () => {
+  for (const slug of allSlugs) {
+    test(`focus-unfocused-${slug}`, async ({ page }) => {
```

Comment:
I'm not sure we need the unfocused tests, they don't seem to bring much value.

I would, however, like to see the tests with dark-charcoal or black background, as that's been a pain point for some focus rings.

Reply:
Dropped the unfocused tests and added one with a dark charcoal background.